seatunnel-dist/src/test/java/org/apache/seatunnel/ImportShadeClassCheck.java

```diff
@@ -0,0 +1,162 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import com.github.javaparser.JavaParser;
+import com.github.javaparser.ParseResult;
+import com.github.javaparser.Range;
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.ImportDeclaration;
+import com.github.javaparser.ast.NodeList;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class ImportShadeClassCheck {
+
+    private static Map<String, NodeList<ImportDeclaration>> importsMap = new HashMap<>();
+    private final String SEATUNNEL_SHADE_PREFIX = "org.apache.seatunnel.shade.";
+    private final boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("win");
+
+    @BeforeAll
+    public static void beforeAll() {
+        Path directoryPath = Paths.get(System.getProperty("user.dir"));
```

Comment:
Does all seatunnel project contains this property?

Reply:
```
2024-11-27T03:20:25.0504611Z [INFO] Running org.apache.seatunnel.api.ImportShadeClassCheckTest
2024-11-27T03:20:25.0505865Z [] 2024-11-27 03:20:25,024 INFO  org.apache.seatunnel.api.ImportShadeClassCheckTest - work directory ===>  D:\a\seatunnel\seatunnel\seatunnel-dist
2024-11-27T03:20:38.9589159Z [] 2024-11-27 03:20:38,920 INFO  org.apache.seatunnel.api.ImportShadeClassCheckTest - check jackson shade successfully
2024-11-27T03:20:39.0284906Z [] 2024-11-27 03:20:39,014 INFO  org.apache.seatunnel.api.ImportShadeClassCheckTest - check guava shade successfully
2024-11-27T03:20:39.1050318Z [] 2024-11-27 03:20:39,092 INFO  org.apache.seatunnel.api.ImportShadeClassCheckTest - check janino shade successfully
2024-11-27T03:20:39.1603157Z [] 2024-11-27 03:20:39,155 INFO  org.apache.seatunnel.api.ImportShadeClassCheckTest - check jetty shade successfully
2024-11-27T03:20:39.1683114Z [INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 14.115 s - in org.apache.seatunnel.api.ImportShadeClassCheckTest
```
we can see from the log that the directory for `System.getProperty("user.dir") ` is  `D:\a\seatunnel\seatunnel\seatunnel-dist` ,  the scan directory is `D:\a\seatunnel\seatunnel\`